packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -4833,6 +4837,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
             <Aggregations
               settings={aggregationSettings}
               isRollup={isRollup}
+              isEditable={isEditableGridModel(model) && model.isEditable}
```

Comment:
Ahh I see, you're passing in `isEditable`, referring to the grid being editable. From within the context of the `Aggregations` component, that doesn't really make sense; see my suggestion above about changing it to `availablePlacements` or `isPlacementDisabled`, or `disabledPlacements`, something like that.